docker/ubuntu-based/vpp/Dockerfile

```diff
@@ -17,6 +17,16 @@ RUN wget -O go.tgz "https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz"
  && tar -C /usr/local -xzf go.tgz \
  && rm go.tgz
 
+# install ETCD
+ENV ETCD_VER=v3.3.9
+ENV GOOGLE_URL=https://storage.googleapis.com/etcd
+ENV GITHUB_URL=https://github.com/coreos/etcd/releases/download
+ENV DOWNLOAD_URL=${GOOGLE_URL}
+
+RUN curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+RUN tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp/etcd --strip-components=1
+RUN rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+
```

Comment:
Installation of ETCD should not be part of this image build process - it's purpose is to build VPP only.

Reply:
Leftovers from testing, shouldn't be there since I moved to the quay.io/coreos/etcd:v3.1.10 removing....